data/human/human missions.txt

```diff
@@ -2502,6 +2502,123 @@ event "terraforming follow-up"
+	source "Var' Kayi"
+	destination "Vara K'chrai"
+	passengers 2
+	to offer
+		has "Terraforming 14: done"
+	on offer
+		conversation
+			`You fly across large swathes of the planet's surface, giving Amy a view of all the greenery and the oceans here. Meanwhile, Tema'a Iriket describes the condition of this world when the Wanderers first began working on it. "The atmosphere was [close to, nearing] a [runaway, uncontrolled] greenhouse effect. Many species had recently gone extinct, or were very close to it. The oceans were also quite toxic. Now, the atmosphere is [stable, sustainable], and many of the species that were [on the brink?] of extinction now flourish."`
+			`	Amy also explains about her recent experimental terraforming. "We believe the planet used to be temperate, or even tropical millions of years ago. We used an asteroid impact to trigger a supervolcano that was close to eruption. The greenhouse gases from the eruption have already begun to warm the climate."`
+			`	Iriket thinks for a moment before responding. "Using the planet's [features, assets] to restore more habitable conditions is an [admirable?] approach." The approval buoys Amy, but she is somewhat deflated by what comes next. "However, an asteroid would not be our [preferred, first] step."`
+			`	As you conclude your sightseeing, Tema'a Iriket says your next stop should be <planet>, where she will provide Amy with more detailed records of their terraforming activities than she can provide orally.`
+			`	Once Iriket has returned to her bunk, Amy says to you, "I wonder what my professor would say about 'anthropomorphizing planets' if he saw this," she gestures towards the images of <origin> on the monitors, "and you told him they believe planets have their own identities, too."`
+				accept
+	on complete
+		dialog "When you arrive on <planet>, Tema'a Iriket tells you to meet her in the spaceport in a few hours before heading off to prepare the data for Amy."
+
+
+
+mission "Terraforming 16"
+	name "Field Trip"
+	description "Return Amy to <destination> following her trip to meet the Wanderers and learn about how they go about terraforming."
+	source "Vara K'chrai"
+	destination "Tundra"
+	passengers 1
+	to offer
+		has "Terraforming 15: done"
+	on offer
+		conversation
+			`Tema'a Iriket meets you in the spaceport and hands Amy a tablet, explaining, "I have stored [records, histories] of over a hundred worlds from our archive here. I hope the translation is [adequate, acceptable]. There has not been much opportunity to test our translation systems for the human language."`
+			`	Amy has already begun browsing the data; Iriket's concerns about the quality of the translation seem to go unheard. "This is incredible. Thank you so much."`
+			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, Captain <last>." Iriket bids you goodbye.`
```

Comment:
```suggestion
			`	"Explaining our work to an outsider has been [gratifying, novel]. I must thank you for the opportunity, Captain <last>," Iriket bids you goodbye.`
```
Should it be done this way, to avoid stubs?

Reply:
I'm not sure what you mean; "Iriket bids you goodbye" is a complete sentence.